### PR TITLE
New Article: Neural Style Transfer 

### PR DESCRIPTION
**Pull request by dharmicksai**
In this blog, I tried to explain component by component of the working of a Style Transfer Model which produces a styled image given  style and content image

### CODE REVIEW

File: _posts/2021-01-07-Neural-style-transfer.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+ 
+![content loss from Gats et al](/blog/assets/img/Neural-Style-Transfer/content_Loss.png)
+ 
+Style loss given A<sup>l</sup> is target style representationof layer l:
+ 
+![style loss per layer from Gats et al](/blog/assets/img/Neural-Style-Transfer/style_loss1.png)
+ 
+![Total style loss from Gats et al](/blog/assets/img/Neural-Style-Transfer/style_loss2.png)
+ 
+Total Loss given content representation p, style representation a, styled image representation x:
```
Comment (geekswaroop) [inline _posts/2021-01-07-Neural-style-transfer.md:47]: Use latex here.

File: _posts/2021-01-07-Neural-style-transfer.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
...code style into an embedding and a common image transformation network for multiple styles. To understand about Instance Normalisation please look at this blog [link](https://medium.com/techspace-usict/normalization-techniques-in-deep-neural-networks-9121bf100d8). We see in this [paper](https://arxiv.org/pdf/1607.08022.pdf) that Instance Normalisation is better than Batch Normalisation in Image Transformation Network. For using a single Image Transformation network for multiple styles, we see the work in this [paper](https://arxiv.org/pdf/1610.07629.pdf), where they found a surprising fact about role of normalisation in Transformation network, where scaling and shifting of the instance normalized activations can be used to produce different styles. For each style, the activations of instances normalised layer are scaled and shifted differently and these scaling and shifting parameters represent the style as an embedding in space, we can use this property and can find embedding of different styles and during production for different styles, we can just change these shifting and scaling parameters in order to produce different styles, which majority of the weights remaining the same.
```
Comment (geekswaroop) [inline _posts/2021-01-07-Neural-style-transfer.md:61]: Uniform capitalisation.

File: _posts/2021-01-07-Neural-style-transfer.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+![Conditional Instance Normalisation](/blog/assets/img/Neural-Style-Transfer/Instance_Normalisation.png) where µ and σ are x’s mean and standard deviation.
+ 
+So for each style we can train γ<sub>s</sub> and β<sub>s</sub> which represents the particular style and can be replaced if we wanted different style. this decreasing the cost of memory.
+ 
+![Applying Instance Normalisation](/blog/assets/img/Neural-Style-Transfer/Normalisation.png)
+ 
+The Image Transformation Network Details are given in this [paper](https://arxiv.org/pdf/1610.07629.pdf) For implementing Multi-style Transfer
```
Comment (geekswaroop) [inline _posts/2021-01-07-Neural-style-transfer.md:68]: Uniform capitalisation.

**Review by geekswaroop — CHANGES_REQUESTED** [review]

File: _posts/2021-01-07-Neural-style-transfer.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "Why GraphQL is a big thing"
```
[the remark this answers is not in the document]
Comment (dharmicksai) [reply _posts/2021-01-07-Neural-style-transfer.md:3]: sorry my bad will correct it

**Comment by dharmicksai** [comment]
hey, I added latex and fixed the typo. Please do look again

**Review by geekswaroop — APPROVED** [review]
LGTM, Good Job on the article :)

**Pull request merged**